data/events/events.xml

```diff
@@ -35,6 +35,10 @@
 	<event class="Player" method="onGainSkillTries" enabled="1" />
 	<event class="Player" method="onWrapItem" enabled="1" />
 	<event class="Player" method="onInventoryUpdate" enabled="1" />
+	<event class="Player" method="onStorageUpdate" enabled="1" />
+	<event class="Player" method="onQuestTracker" enabled="1" />
+	<event class="Player" method="onQuestLog" enabled="1" />
+	<event class="Player" method="onQuestLine" enabled="1" />
```

Comment:
I would change these event names to be more explicit about what they are reacting to:

```suggestion
	<event class="Player" method="onRequestQuestTracker" enabled="1" />
	<event class="Player" method="onRequestQuestLog" enabled="1" />
	<event class="Player" method="onRequestQuestLine" enabled="1" />
```

Or `open` instead of `request`. All other events are (or should be) in the format `on<action><subject>`

Also, `onStorageUpdate` -> `onUpdateStorage`. `onInventoryUpdate` is incorrect.